peopleops/index.md

```diff
@@ -0,0 +1,17 @@
+## Vacation Policy
+
+FlowForge has a unlimited time off policy. Taking vacation is encouraged for all
+team members. To prevent an undefined number of vacation days to start a race to
+the bottom, we recommend each team member to take a minimum of 25 days a year,
+and at least 5 days a quarter.
+
+### Logging time off
+
+When taking more than 3 days off consecutively, tell (don't ask) your manager. 
```

Comment:
@ZJvandeWeg should this line read "When taking 3 or less days off consecutively, tell (don't ask) your manager. " ?

Reply:
Sorry, a 2nd read I guess the whole thing is something like...

* When taking 3 or less days off consecutively, there is no need to inform your manager
* When taking more than 3 days off consecutively, tell (don't ask) your manager
* When taking over 2 weeks off consecutively, seek approval from your manager

... is that correct?

Reply:
I think even for a day off, it should be a common courtesy to let your team/manager know (where possible)

Reply:
@Steve-Mcl Yes, the 2nd read was my intent. 

@knolleary Agree, though I want everyone to feel the liberty to decide that today is a day off without needing to explain them self to their manager. Further, given our values contains a results section, it shouldn't matter as long as folks stay effective and get results for the company. Happy to update the wording, do you have a suggestion?

Reply:
@knolleary I'm merging this as is mostly to unblock another PR. As always we can extend or reword in a PR later.